rust/src/lib.rs

```diff
@@ -112,7 +113,7 @@ pub enum Event {
 #[derive(Debug, Clone)]
 pub struct Metadata {
```

Comment:
If this is now just for passing around TelemetryID I think the event name should reflect that.

Reply:
I'll record that for follow up